src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -94,18 +95,23 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
+
     result_poller = client.import_image(
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
 
+    if no_wait:
+        return result_poller
```

Comment:
Simply returning the poller might not be the best way to go about this. 

The [authoring command modules](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md) section on supporting `--no-wait`

Mentions the cli core method `azure.cli.core.util.sdk_no_wait` defined here in [util.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/util.py#L646).

```python
sdk_no_wait(no_wait, func, *args, **kwargs)
```
This method does calls the function you pass with the supplied args and kwargs, and sets/updates the arguments to include `polling: False`. 

Given our code doesn't rely on sdk_no_wait, some approaches come to mind on how we can either use it or copy the behavior of sdk_no_wait when handling result.